### ISSUE TITLE
Arbitrary ConstraintPriority?

### ISSUE THREAD

**Issue by radianttap** [roberthein/TinyConstraints#28]
What is the purpose of `ConstraintPriority`, meaning why it exists?
It's used in all the methods but is limited to a handful of values which is really limiting. I often need to use values like 999 to resolve issues when container is temporary smaller than content (but will be fine at the end of the animation).

For example, this is impossible:

```
v.edges(to: pv, priority: ConstraintPriority(rawValue: UILayoutPriority(999))!)
```

**Comment by roberthein** [comment]
I think you have a valid point, I could add a **custom** `case`, like this for example:
`v.edges(to: pv, priority: .custom(999))`
What do you think?

**Comment by radianttap** [comment]
It seems that could work, simply applying the associated value as UILayoutPriority.

**Comment by roberthein** [comment]
Ok I'll add this option to *TinyConstraints*. 👍 Thank you! 🙏 
In the meanwhile you can adjust the `priority` like this:
```
let constraints = v.edges(to: pv)
constraints.forEach {
    $0.priority = UILayoutPriority(rawValue: 999)
}
```

**Comment by radianttap** [comment]
I just implemented the workaround and it's a bit different:

```
let constraints = v.edges(to: pv, isActive: false)
constraints.forEach {
	$0.priority = UILayoutPriority(999)
	$0.isActive = true
}
```

Reason for isActive dance is this:

> *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', 
> reason: 'Mutating a priority from required to not on an installed constraint (or vice-versa) is not supported.  
> You passed priority 999 and the existing priority was 1000.'
>

**Comment by roberthein** [comment]
Ah ok, yes that makes sense, sorry. I will update **TinyConstraints** soon so you can get rid of this workaround. ✌️

**Comment by roberthein** [comment]
This has been fixed in 3.0
